db/seeds.rb

```diff
@@ -5,3 +5,6 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+require 'factory_bot'
+
+FactoryBot.create_list(:company, 50)
```

Comment:
add new line at end of file

Reply:
> add new line at end of file

most of the code editors do this for you automatically, just turn this feature on in settings